// Copyright (c) 2023 Boston Dynamics, Inc.  All rights reserved.
//
// Downloading, reproducing, distributing or otherwise using the SDK Software
// is subject to the terms and conditions of the Boston Dynamics Software
// Development Kit License (20191101-BDSDK-SL).

syntax = "proto3";

package bosdyn.api.auto_return;

option java_outer_classname = "AutoReturnServiceProto";

import "bosdyn/api/auto_return/auto_return.proto";

service AutoReturnService {

    // Configure the service.
    rpc Configure(ConfigureRequest) returns (ConfigureResponse);

    // Get the current configuration.
    rpc GetConfiguration(GetConfigurationRequest) returns (GetConfigurationResponse);

    // Start AutoReturn now.
    rpc Start(StartRequest) returns (StartResponse);
}
